src/main/java/slimeknights/tconstruct/common/data/tags/BlockTagProvider.java

```diff
@@ -503,6 +503,7 @@ private void addHarvest() {
     // melting pan blacklist, basically anything that feels gross due to unsupported melting recipe
     tagBlocks(MINEABLE_MELTING_BLACKLIST, TinkerSmeltery.searedMelter, TinkerSmeltery.smelteryController, TinkerSmeltery.foundryController, TinkerSmeltery.searedLantern, TinkerSmeltery.scorchedLantern, TinkerSmeltery.searedFluidCannon, TinkerSmeltery.scorchedFluidCannon);
     tagBlocks(MINEABLE_MELTING_BLACKLIST, TinkerSmeltery.searedTank, TinkerSmeltery.scorchedTank);
+    tagBlocks(MINEABLE_MELTING_BLACKLIST, TinkerSmeltery.searedCastingTank);
```

Comment:
Just add this to the end of the line with scorched fluid cannon. Reason there are two calls is the first is over standard `ItemObject` while the second is over `EnumObject`